views/templates/Booklet/edit.tpl

```diff
@@ -4,12 +4,22 @@ Template::inc('form_context.tpl', 'tao');
 ?>
 <?= tao_helpers_Scriptloader::render() ?>
 <header class="section-header flex-container-full">
-    <h2><?=get_data('formTitle')?></h2>
+    <h2><?=get_data('form-title')?></h2>
 </header>
 <div class="main-container flex-container-main-form">
     <div class="form-content">
-        <?=get_data('myForm')?>
+        <?=get_data('form-fields')?>
     </div>
 </div>
 
 <?php Template::inc('footer.tpl', 'tao'); ?>
+
+<script>
+    requirejs.config({
+        config: {
+            'taoBooklet/controller/Booklet/editBooklet' : {
+                'isPreviewEnabled' : <?= json_encode(get_data('isPreviewEnabled')) ?>
```

Comment:
Perhaps, it would be better to set an associative array of config values here, just in case we'd need to add more in the future. Then you won't have to add more and more `get_data()` calls in the template.

Reply:
Fixed